static/app/components/feedback/useFetchFeedbackInfiniteListData.tsx

```diff
@@ -41,17 +41,19 @@ export const EMPTY_INFINITE_LIST_DATA: ReturnType<
   issues: [],
   loadMoreRows: () => Promise.resolve(),
   setFeedback: () => undefined,
+  setQueryStatus: () => undefined,
 };
 
 export default function useFetchFeedbackInfiniteListData({queryView}: Params) {
   const organization = useOrganization();
+  const [queryStatus, setQueryStatus] = useState('unresolved');
```

Comment:
i think we should followup on this and put the state into the url. So that way if someone wants to share what they're looking at the correct mailbox will be picked